cs/test/core/Util.cs

```diff
@@ -999,12 +999,16 @@ public static IBonded<T> MakeBondedSP<T>(T obj)
                 if (!AnyField<From>(Reflection.IsBonded))
                 {
                     streamTranscode(SerializeSP, TranscodeSPXml<From>, DeserializeXml<To>);
-                
-                    // NewtonSoft JSON doesn't support uint64
+
+#if SUPPORTS_BIGINTEGER
```

Comment:
This runs the transcoding twice when we do have BigInteger support. What do you think about structuring it this way?

    #if SUPPORTS_BIGINTEGER
        const bool hasBigInteger = true;
    #else
        const bool hasBigInteger = false;
    #endif

    // NewtonSoft JSON doesn't support uint64 in portable profile, so we skip that type
    if (hasBigInteger || (typeof (From) != typeof (MaxUInt64)))
    {
        streamTranscode(SerializeSP, TranscodeSPJson<From>, DeserializeJson<To>);
    }

And a similar condition later on for the if that guards to tests around line 1039.